x-pack/plugin/core/src/main/java/org/elasticsearch/license/XPackLicenseState.java

```diff
@@ -391,8 +372,8 @@ public synchronized boolean isAuditingAllowed() {
      *
      * @return true if the license allows for the stats and health APIs to be used.
      */
-    public synchronized boolean isStatsAndHealthAllowed() {
-        return status.active;
+    public boolean isStatsAndHealthAllowed() {
+        return isActive();
```

Comment:
`isActive` is an existing synchronized method.

Reply:
I'd prefer a method name like `allowForAllLicenses()`

Most of these `isXYZAllowed()` methods are added by people who deal with licensing questions _very rarely_. They just want to come in and implement a method in the most obvious way.  
They don't know what "active" means (_I_ forget what it means half the time), and their likely response here is "I'll just copy one of these other methods and hope that it's right". And it will be, but I'd rather we had something that was _clearly_ correct to them, so that they don't need to hope. And that is a method that is named to be exactly in line with their requirement.  
There's a good chance the PR reviewer isn't an expert on licensing either, so a method that copies this code isn't obviously correct to the reviewer - they need to look at how other methods are implemented to check. A method that has the equivalent of `isMyFeatureAllowed() { return thisIsAllowedForAllLicenseTypes(); }` is correct just from reading the diff.

I'm very open to discussion on a preferred name, but I'm pretty keen for these to be readable in a _literal_ sense.